fahadmak/stackoverflow_clone
# #167138075 Create question tags functionality

**What does this PR do?**
This PR ensures that a user can create question tags on the front page.
**Description of Task to be completed?**
- Create Tag model
- Add tag field in Question model

**What are the relevant Pivotal Tracker stories?**
[#167138075](https://www.pivotaltracker.com/n/projects/2364284/stories/167138075)

**How should be tested manually?**
Run each of the following commands in your terminal in the respective order.

- `git clone https://github.com/fahadmak/stackoverflow_clone.git`
- `cd stackoverflow_clone`
- `git checkout ft-create-list-question-comments-167138181`
- `virtualenv venv`
- `. venv/bin/activate`
- `pip install -r requirements.txt`
- `run migrations using this command python manage.py migrate`
- `run python manage.py runserver 127.0.0.1:8000`
- `Go to this link in your browser 127.0.0.1:8000`
- `You will be redirected to the login page where you should log in`
- `Click on the 'Register Today' link to go to sign up page`
- `Try to sign up an account and then use it in the login page`
- `Create a question and add a tag`

**Screenshots**
<img width="1021" alt="Screen Shot 2019-07-26 at 9 23 29 AM" src="https://user-images.githubusercontent.com/13882936/61930648-0de3f600-af87-11e9-807b-9067c5de225f.png">
